init_configure_commands_tut.md

```diff
@@ -0,0 +1,156 @@
+<a href="https://aws-amplify.github.io/" target="_blank">
+    <img src="https://s3.amazonaws.com/aws-mobile-hub-images/aws-amplify-logo.png" alt="AWS Amplify" width="550" >
+</a>
+
+# AWS Amplify CLI initialization & configuration tutorial
+
+The AWS Amplify CLI helps you set up an AWS IAM user, set up an AWS profile on your local system and the initialize Amplify powered apps with all your defined configurations. We'll also walk through steps as to how you could share the Amplify generated backend configuration between other frontend projects and re-use the same backend infrastructure in-between multiple platforms/users.
+
+In this tutorial, we'll go walk through setting up an Amplify enabled project and go in detail as to what happens behind the scenes when doing so.
+
```

Comment:
Can you add a TOC here with hyperlinks to the 3 areas? You just need to use a [Title](#anchor) format.